src/k8s/pkg/k8sd/setup/kube_apiserver.go

```diff
@@ -43,7 +45,7 @@ var (
 )
 
 // KubeAPIServer configures kube-apiserver on the local node.
-func KubeAPIServer(snap snap.Snap, serviceCIDR string, authWebhookURL string, enableFrontProxy bool, datastore string, authorizationMode string) error {
+func KubeAPIServer(snap snap.Snap, serviceCIDR string, authWebhookURL string, enableFrontProxy bool, datastore string, datastoreUrl string, authorizationMode string) error {
```

Comment:
Rename `datastoreUrl` to `externalDatastoreURL` (also note caps), it is unused if `datastore == "k8s-dqlite"`